lib/util/git/conflicts-cache.ts

```diff
@@ -1,56 +1,42 @@
+import { logger } from '../../logger';
 import { getCache } from '../cache/repository';
 
 export function getCachedConflictResult(
-  targetBranchName: string,
-  targetBranchSha: string,
-  sourceBranchName: string,
-  sourceBranchSha: string
+  branchName: string,
+  branchSha: string | null,
+  baseBranch: string,
+  baseBranchSha: string | null
```

Comment:
When would the SHAs be null?

Reply:
Did this for consistent typing.

Reply:
we shouldn't allow null if thesly never should be null

Reply:
https://github.com/renovatebot/renovate/blob/0a44eb78a2e8b8b12611a06f4ae4a22e9f22ebd6/lib/util/git/index.ts#L698-L699

SHAs returned from `getBranchCommit` maybe null so the 2 asn since we use this fn to get our SHAs I could think of two ways to handle this without causing #7154 
1. if statement inside `isBranchConflicted` or 
2. adding `null` type to cached fns SHA params.

Reply:
I thought if null is returned then it means we can't find the branch and should not be continuing into the cache checks anyway?

Reply:
We return true in such cases. 
There is already a null check here so the tyep I added is unnecessary
https://github.com/renovatebot/renovate/blob/4d8e2c704f15b37140f671601cf361c931d28421/lib/util/git/index.ts#L698-L706